schema/features/network-protection.ts

```diff
@@ -0,0 +1,30 @@
+import { Feature, SubFeature } from '../feature';
+
+// Type of the feature `settings` object
+type SettingsType = any;
```

Comment:
Maybe the following works?
```suggestion
type SettingsType = {
    autoExcludeApps: {
      packageName: string;
    }[];
  };
```